relayer/relays/parachain/config.go

```diff
@@ -9,17 +9,18 @@ type Config struct {
 	Sink   SinkConfig   `mapstructure:"sink"`
 }
 
+// TODO: check whether ChannelID should be uint32 (as in the parachain) or big.Int (uint256, as in the Gateway contract)
```

Comment:
@vgeddes How big are the channelIDs?

Reply:
Guess uint32 as ParaId should be enough. But remember Aidan mentioned last time we'll probably allow multiple channels for each Parachain?

Reply:
Ah right, we can expand the channel id on Polkadot to get multiple channels per parachain